tests/cli/delete/test_cli_delete_file.py

```diff
@@ -64,3 +67,44 @@ def test_delete_existing_file_no_confirmation(
 
     # THEN file delete should be in output
     assert "file deleted" in caplog.text
+
+
+def test_delete_sample_sheet(
+    populated_context: Context, cli_runner: CliRunner, caplog, flow_cell_id: str, sample_sheet: Path
+):
+    # GIVEN a bundle with a file tagged with sample-sheet
+    store: Store = populated_context["store"]
+    assert store.get_files(file_path=str(sample_sheet)).one()
+
+    # WHEN deleting the sample sheet
+    cli_runner.invoke(
+        delete.delete_sample_sheet, [str(flow_cell_id), "--yes"], obj=populated_context
+    )
+
+    # THEN the sample sheet should be deleted
+    assert not store.get_files(file_path=str(sample_sheet)).first()
+
+
+def test_delete_sample_sheet_included(
+    populated_context: Context, cli_runner: CliRunner, caplog, flow_cell_id: str, sample_sheet: Path
+):
+    # GIVEN an included bundle with a file tagged with sample-sheet
+    store: Store = populated_context["store"]
+    sample_sheet_file: File = store.get_files(file_path=str(sample_sheet)).one()
+    include_version(global_root=populated_context["root"], version_obj=sample_sheet_file.version)
+    assert sample_sheet_file
+    assert sample_sheet_file.is_included
+    assert Path(sample_sheet_file.full_path).exists()
+
+    # WHEN deleting the sample sheet
+    cli_runner.invoke(
+        delete.delete_sample_sheet,
+        [flow_cell_id, "--yes"],
+        obj=populated_context,
```

Comment:
As in the test above
```suggestion
        delete.delete_sample_sheet, [flow_cell_id, "--yes"], obj=populated_context,
```